### PR TITLE
update review limit's name

### PR DESCRIPTION
**Pull request by brishtibheja**
Noticed we didn't change review limit's name here.

### CODE REVIEW

File: ftl/core/deck-config.ftl [hunk 1]
```diff
@@ -16,7 +16,7 @@ deck-config-title = Deck Options
 
 deck-config-daily-limits = Daily Limits
 deck-config-new-limit-tooltip =
-    The maximum number of new cards to introduce in a day, if new cards are available.
```
Comment (dae) [inline ftl/core/deck-config.ftl:19]: Not sure about this - the new card limit is a maximum, not a quota.

**Comment by brishtibheja** [comment]
@dae I didn't realise 'maximum' made a difference in meaning until now.

**Comment by dae** [comment]
Thanks!

**Comment by brishtibheja** [comment]
@dae you forgot to revert this and this: https://github.com/ankitects/anki/pull/3326

**Comment by dae** [comment]
Thanks :-)

**Pull request merged**